src/Datadog.Trace.Owin/OwinTags.cs

```diff
@@ -0,0 +1,19 @@
+using System.Linq;
+using Datadog.Trace.ExtensionMethods;
+using Datadog.Trace.Tagging;
+
+namespace Datadog.Trace.Owin
+{
+    internal class OwinTags : WebTags
+    {
+        private const string ComponentName = "Owin";
+
+        private static readonly IProperty<string>[] AspNetCoreTagsProperties =
+            WebTagsProperties.Concat(
+                new ReadOnlyProperty<OwinTags, string>(Trace.Tags.InstrumentationName, t => t.InstrumentationName));
+
+        public string InstrumentationName => ComponentName;
```

Comment:
Not necessary for in this PR, but we should probably promote "component" to a top-level tag at some point and make sure it's included in all integrations.